apps/demos/project.json

```diff
@@ -12,19 +12,14 @@
     "build": {
       "executor": "@angular-devkit/build-angular:browser",
       "options": {
-        "outputPath": "docs/demos",
+        "outputPath": "dist/apps/docs/demos",
```

Comment:
The build output directory for the `docs` project is now `dist/apps/docs` instead of the raw files in `docs`.